Dockerfile

```diff
@@ -20,6 +20,7 @@ RUN apt-get update && \
 
 RUN go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.28.1 && \
         go install github.com/twitchtv/twirp/protoc-gen-twirp@v8.1.3+incompatible && \
-        go install github.com/github/twirp-ruby/protoc-gen-twirp_ruby@v1.10.0
+        go install github.com/github/twirp-ruby/protoc-gen-twirp_ruby@v1.10.0 && \
```

Comment:
is there some documentation for using this grpc?

Reply:
yep when we cut a new release of this after merge, anyone using it should be able to use the `--go-grpc_out` and `--go-grpc_opt` flags in `copilot-api` by upgrading to the new release

https://grpc.io/docs/languages/go/quickstart/#regenerate-grpc-code